rio/ChangeLog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog for rio
 
+## 0.1.10.0
```

Comment:
I haven't actually released 0.1.9.0 yet, so if you'd like, we can include this change there.